Evgeny_Khrapan/02.GIT.Local/02.GIT.Local.md

```diff
@@ -0,0 +1,169 @@
+# Commands list from history
+  
+
```

Comment:
Could you please use code section markup  to display code/shell commands

Reply:
fixed